### PR TITLE
Bailey changes

### PR DESCRIPTION
**Pull request by CodeMonster4**
Merging changes.

### CODE REVIEW

**Comment by CodeMonster4** [comment]
@Enigmatic-Star I can add comments clarifying things.

File: dir1/app1.js [hunk 1]
```diff
@@ -1 +1,5 @@
-console.log(123);
\ No newline at end of file
+console.log(123);
+
+var baileyChanges = 1
+var moreChanges = 2
+console.log(baileyChanges + moreChanges)
```
Comment (CodeMonster4) [inline dir1/app1.js:5]: @Enigmatic-Star I can also add inline comments here.

Comment (CodeMonster4) [reply dir1/app1.js:5]: `
var newCodeToAdd = "Let's add some more code\n"
`

**Review by Enigmatic-Star — APPROVED** [review]
changes approved

**Pull request merged**